tests/integration_tests/build/test_licenses.py

```diff
@@ -10,22 +10,25 @@
 AMAZON_COPYRIGHT_YEARS = (2018, 2019)
 AMAZON_COPYRIGHT = (
     "Copyright {} Amazon.com, Inc. or its affiliates. All Rights Reserved."
-    )
-AMAZON_LICENSE = (
-        "SPDX-License-Identifier: Apache-2.0"
-    )
+)
+APACHE_LICENSE = (
+    "SPDX-License-Identifier: Apache-2.0"
+)
 CHROMIUM_COPYRIGHT = (
     "Copyright 2017 The Chromium OS Authors. All rights reserved."
-    )
+)
 CHROMIUM_LICENSE = (
     "Use of this source code is governed by a BSD-style license that can be"
-    )
+)
+SUSE_COPYRIGHT = (
```

Comment:
Where is the SUSE copyright added?

Reply:
I added `tests/integration_tests/functional/test_initrd.py`. If I can put the initrd tests in an already copyrighted file,  then this is not needed.